### PR TITLE
Added melting for ice blocks

### PR DESCRIPTION
**Pull request by thebigsmileXD**
onUpdate checks for lightlevel 12 or higher at the ice position and "melts" (replaces it with water) it

### CODE REVIEW

**Comment by shoghicp** [comment]
Can you fix your formatting so it follows the [Contribution Guidelines](https://github.com/PocketMine/PocketMine-MP/blob/master/CONTRIBUTING.md), then force-push to your branch the amended commit?

**Comment by thebigsmileXD** [comment]
Actually.. i looked into it, i cant see any formatting problems

File: src/pocketmine/block/Ice.php [hunk 1]
```diff
@@ -53,4 +53,14 @@ public function onBreak(Item $item){
 	public function getDrops(Item $item){
 		return [];
 	}
-}
\ No newline at end of file
+	
+	public function onUpdate($type){
+		if($type === Level::BLOCK_UPDATE_RANDOM){
+			if($this->getLevel()->getBlockLightAt($this->x, $this->y, $this->z)>= 12){
+			$this->getLevel()->setBlock($this, new Water(), true);
```
Comment (sekjun9878) [inline src/pocketmine/block/Ice.php]: For example, this code should be indented.

**Comment by haruajsuru** [comment]
you forgot to add: 
use pocketmine\level\Level;

**Comment by thebigsmileXD** [comment]
Finisched.

**Comment by thebigsmileXD** [comment]
#3249

**Comment by dktapps** [comment]
This is obsoleted by changes in the PMMP repository.

**Pull request closed without merging**